hazelcast/cluster.py

```diff
@@ -58,9 +68,11 @@ class ClusterService(object):
     """
 
     def __init__(self, internal_cluster_service):
+        # type: (Any) -> None
```

Comment:
As the name suggests, `internal_cluster_service` is the type of `_InternalClusterService` not `Any`